e2e/__tests__/detectOpenHandles.ts

```diff
@@ -72,9 +72,13 @@ it('does not report promises', () => {
 });
 
 describe('notify', () => {
-  skipSuiteOnWindows();
-
   it('does not report --notify flag', () => {
+    if (process.platform === 'win32') {
```

Comment:
This is the only test aside from the yarn pnp that is now being skipped on Windows 🎉 

Open to ideas on if anything can be done to eliminate it - I suspect it represents an actual "bug" (/ maybe just something that can't be supported?)